src/main/java/common/RatType.java

```diff
@@ -5,8 +5,9 @@
  *
  */
 public enum RatType {
-	LTE("LTE"),
-	WCDMA("WCDMA");
+	
+    LTE("Long-Term Evolution"),
+	WCDMA("Wideband Code Division Multiple Access");
```

Comment:
When Ista assumed we all meant what WCDMA stood for I felt like I didn't belong as an Ericsson employee.